src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.cpp

```diff
@@ -180,6 +180,26 @@ void JournalFileProcessor::process()
                 iter->recordIndex(),
                 iter->recordOffset());
         }
+        else if (iter->recordType() == mqbs::RecordType::e_QUEUE_OP) {
+            const mqbs::QueueOpRecord& record = iter->asQueueOpRecord();
+
+            stopSearch = d_searchResult_p->processQueueOpRecord(
+                record,
+                iter->recordIndex(),
+                iter->recordOffset());
+        }
+        else if (iter->recordType() == mqbs::RecordType::e_JOURNAL_OP) {
+            const mqbs::JournalOpRecord& record = iter->asJournalOpRecord();
+
+            stopSearch = d_searchResult_p->processJournalOpRecord(
+                record,
+                iter->recordIndex(),
+                iter->recordOffset());
+        }
+        else {
+            stopSearch = d_searchResult_p->processOtherRecord(
```

Comment:
I believe it's valuable and important track (and show in the output) the records for which there are no specific processing code.
Previously it would work for JOUNRAL_OP and QUEUE_OP. And in future if new record types are added, we'll see them immediately in the StorageTool output without extra effort.

Generally I believe it is more consistent to have 'else' in this case to take into account all the records from the Journal file.